lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -314,6 +321,12 @@ def add_creditcard(post, creditcard, options)
         end
       end
 
+      def add_emv_creditcard(post, icc_data, options = {})
+        card = {}
+        card[:emv_auth_data] = icc_data
+        post[:card] = card
```

Comment:
A lot was happening in this method before, so this is the residue I guess. ;) I'll change it.